src/condor_includes/condor_crypt.h

```diff
@@ -26,6 +26,27 @@
 
 #include "CryptKey.h"
 
+struct StreamCryptoState {
+    // The IV is split in two: a 32-bit counter and a
+    // 64-bit random number for a total of 12 bytes.
+    // The ctr is added to the last 4 bytes of the IV.
+    union Packed_IV {
```

Comment:
Now that we don't pack the IV, let's simply use a struct and drop the funky union.

Reply:
Talked with Zach about this out-of-band -- decision was to keep the "packed" approach.  Resolving item for now.